build.Tests.ps1

```diff
@@ -68,7 +68,8 @@ Describe $script {
     }
     It "shows a specific dsc_psrepository resource" {
       # PSGallery is the default repo always installed
-      $Result = Invoke-PdkCommand -Path $expected_base -Command 'pdk bundle exec puppet resource dsc_psrepository title dsc_name=PSGallery' -PassThru -SuccessFilterScript {
+      $Result = Invoke-PdkCommand -Path "$expected_base/powershellget" -Command 'pdk bundle exec puppet resource dsc_psrepository title dsc_name=PSGallery' -PassThru -SuccessFilterScript {
```

Comment:
Does this line still need to be changed to pass the tests?

Reply:
Sorry was left over from my testing I was trying. I have now removed my 'testing' commit.